scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -1287,25 +1290,27 @@ class FormatOps(
       else tokens(nextNonComment(maybeArrow), 1)
     }(x => prevNonComment(tokens(x, -1)))
 
+  case class TreeArgs(tree: Tree, args: Seq[Tree])
```

Comment:
i'd probably define this class outside of `FormatOps` class, as it doesn't need to refer to the `FormatOps` instance; can put it inside companion `object FormatOps` if you'd like